tests/integration_tests/security/demo_seccomp/Cargo.lock

```diff
@@ -1,3 +1,5 @@
+# This file is automatically @generated by Cargo.
```

Comment:
How'd this end up in the PR? I think these Cargo.lock comments appeared after a rustc upgrade, but this crate shouldn't be built as part of the "normal" build process; is it a leftover from running tests outside the container? In either case, please revert & remove it from this PR